tool/tctl/common/auth_command.go

```diff
@@ -70,7 +71,9 @@ func (a *AuthCommand) Initialize(app *kingpin.Application, config *service.Confi
 	a.authSign.Flag("user", "Teleport user name").StringVar(&a.genUser)
 	a.authSign.Flag("host", "Teleport host name").StringVar(&a.genHost)
 	a.authSign.Flag("out", "identity output").Short('o').StringVar(&a.output)
-	a.authSign.Flag("format", fmt.Sprintf("identity format: %q (default), %q, or %q", client.IdentityFormatFile, client.IdentityFormatOpenSSH, client.IdentityFormatTLS)).Default(string(client.DefaultIdentityFormat)).StringVar((*string)(&a.outputFormat))
+	a.authSign.Flag("format", fmt.Sprintf("identity format: %q (default), %q, or %q", identityfile.FormatFile, identityfile.FormatOpenSSH, identityfile.FormatTLS)).
+		Default(string(identityfile.DefaultFormat)).
+		StringVar((*string)(&a.outputFormat))
```

Comment:
Maybe consider reformatting the line below in this same format (dotted argument then newline) too.

Reply:
done